zkevm-circuits/src/table.rs

```diff
@@ -132,7 +132,10 @@ pub struct TxTable {
     /// Index for Tag = CallData
     pub index: Column<Advice>,
     /// Value
-    pub value: word::Word<Column<Advice>>,
+    pub value_word: word::Word<Column<Advice>>,
+    /// Value
+    #[deprecated(note = "value_word is fav")]
+    pub value: Column<Advice>,
```

Comment:
@ed255 @hero78119 

Inspired by @ed255's idea of duplicating the struct, I figured out that we only need to duplicate the fields. We can maintain the legacy fields and work on everything on top of the new fields. We now revert almost no existing progress by @hero78119 and still keep the compatibility.